Makefile

```diff
@@ -1,15 +1,26 @@
-NDK_ROOT := $(HOME)/Library/Android/sdk/ndk/26.1.10909125
-APP_ROOT := $(HOME)/AndroidStudioProjects/DroidCamera
+UNAME=$(shell uname)
+ifeq ($(UNAME),Linux)
+	NDK_ROOT ?= $(HOME)/Android/Sdk/ndk/26.1.10909125
+	HOST_OS ?= linux
+	GOARCH := amd64
+	CC_ARCH ?= x86_64
```

Comment:
[q] Even if we are on x86 linux machine, we still want to target aarch64 android right?

Reply:
on intel laptops they give you an intel emulator so that CPU can run it natively

(you may need to override this if you add CI)